### ISSUE TITLE
Publish new version to forge for puppet 4 compatability

### ISSUE THREAD

**Issue by cwood** [arioch/puppet-ulimit#9]
What is in source code and what is on the forge are two different versions and it looks like your local version as of this issue will work for puppet4. Can you push this up to forge?

**Comment by bschonec** [comment]
Yes, please!  Puppet 4 doesn't like the unquoted case statements in params.pp.

**Comment by Falkor** [comment]
+1: would be great if you can just publish asap the current version on the forge to have it compliant with puppetfiles + puppet 4
